labs/gps_lab/README.md

```diff
@@ -0,0 +1,68 @@
+#GPS LAB
+
+##Description
+GPS Lab uses a Raspberry PI with a GPS sensor to fetch current location (latitude and longitude)
+ and sends the info to IoT hub.
+
+ A web application reads the data from IoT Hub real time and displays the GPS location in a Azure MAP.
+
+ ##Steps
+ ### Create and configure IoT Hub
+ 1. Create one azure IoT Hub and create one device under it 
+ 2. Install azure CLI (https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest)
```

Comment:
Can you remove the locale from all links. Can you also not use raw links, but instead use something like:

following the instructions in the [Azure CLI documentation](https://docs.microsoft.com/cli/azure/install-azure-cli)